src/ProxyClient/VarnishAdminMultiple.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace FOS\HttpCache\ProxyClient;
+
+class VarnishAdminMultiple
+{
+    public function __construct($servers)
+    {
+
+    }
+}
```

Comment:
looking at VarnishAdmin, i wonder if we should have a VarnishCli class that wraps the socket connection. then VarnishAdmin would become a lot simpler, and having multiple Cli instances would become simpler. if we keep it the way we currently have it, having a separate wrapper to multiplex seems better to me.